lib/rules/no-mocha-global-arrows.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+/**
+ * @fileoverview Disallow arrow functions as arguments to Mocha globals
+ * @author Paul Melnikow
+ */
+
+var R = require('ramda'),
+    mochaFunctionNames = [
+        'describe',
+        'describe.skip',
+        'xdescribe',
+        'suite',
+        'context',
+        'xcontext',
+        'specify',
+        'xspecify',
+        'it',
+        'it.only',
+        'xit',
+        'test',
+        'test.only',
+        'specify',
+        'specify.only',
+        'before',
+        'after',
+        'beforeEach',
+        'afterEach'
+    ];
+
+module.exports = function (context) {
+    function isLikelyMochaGlobal(scope, name) {
+        return !R.find(R.propEq('name', name), scope.variables);
+    }
+
+    return {
+        CallExpression: function (node) {
+            var callee = node.callee,
+                fnArg;
+
+            if (callee && callee.name && mochaFunctionNames.indexOf(callee.name) > -1) {
+                fnArg = node.arguments.slice(-1)[0];
```

Comment:
No, that won't work. Consider these examples:

```js
beforeEach(function () { });
beforeEach('prepare', function () { });
```

In the first example, `node.arguments[1]` will be undefined.